console-store/Clases/Product.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace console_store.Clases
+{
+    internal class Product
+    {     
+        public required int id { get; set; }
```

Comment:
For properties/classes/methods use PascalCase, see https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions